### PR TITLE
fixed outline on 'Configuration' panel heading in desktop-gui

### LINKED ISSUE #3357: Settings / Configuration has weird outline
Cypress 3.1.5

- `npx cypress open`
- click on "Settings"
First problem - since there is "Configuration" only - why is it collapsed by default?

- click "Configuration"
There is a blue outline which cuts into the help link and just looks weird.

screen shot 2019-02-06 at 9 25 39 pm

### PR DESCRIPTION
**Pull request by King-Kyle**
Removed default outline from the dropdowns under settings in issue #3357.
WIP. was unable to replicate collapsed configuration.

close #3357

### CODE REVIEW

File: packages/desktop-gui/src/settings/settings.scss [hunk 1]
```diff
@@ -148,6 +148,10 @@
     position: relative;
     left: -15px;
 
+    &:focus {
+      outline: 0;
```
Comment (lilaconlee) [inline packages/desktop-gui/src/settings/settings.scss]: While this fixes the goof, it's [worse for accessibility](https://a11yproject.com/posts/never-remove-css-outlines/). I'd try setting a width on `.rc-collapse-header` so it's still there, but doesn't run into "Learn more".

Comment (King-Kyle) [reply packages/desktop-gui/src/settings/settings.scss]: I should have known better. Thanks for the info, I will amend that.

**Comment by King-Kyle** [comment]
Thanks for the response :). I'd like to give that other issue a try as well...Although using the command 'npm run cypress:open' from the contributing docs doesn't really help with testing in this instance as the Project ID and Record Key panels show up.
My changes don't seem to show when running  'npx cypress open' as well. Do you have any suggestions on how I could test this with the Config panel alone?
Otherwise, I think it could be moved to a separate issue if it's not a problem.

**Comment by lilaconlee** [comment]
To work on the config panel alone, I'd recommend writing some tests in `settings_spec.coffee` that check whether the configuration panel is open or closed by default depending on the presence of project ID/record keys. Looks like [this test](https://github.com/cypress-io/cypress/blob/develop/packages/desktop-gui/cypress/integration/settings_spec.coffee#L243) has a set up where you can replicate only the configuration being shown.

You can find this in the [readme](https://github.com/cypress-io/cypress/tree/develop/packages/desktop-gui), but you'll want to watch the `desktop-gui` files to make sure your changes show up and run the cypress tests from the `desktop-gui` directory.

**Pull request merged**